### PR TITLE
Fix some behat tests

### PR DESCRIPTION
**Pull request by wbloszyk**
This PR includes:
- [x] [fix flash messages](https://github.com/sonata-project/twig-extensions/pull/90)
- [x] fix subsite
- [x] [fix flash messages translations](https://github.com/sonata-project/ecommerce/pull/687)
- [x] fix  ("The "hinclude" renderer does not exist."). under "/qa/page/controller-helper"
- [x] fix some routes in tests

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -271,20 +271,6 @@
                 "redis",
                 "xcache"
             ],
-            "funding": [
```
Comment (jordisala1991) [inline composer.lock:274]: seems like you are using an old composer version

Comment (wbloszyk) [reply composer.lock:274]: Upgraded to: 
Composer version 1.10.10 2020-08-03 11:35:19

from 1.6 :smiley:

File: config/packages/framework.yaml [hunk 2]
```diff
@@ -17,3 +17,4 @@ framework:
 
     templating:
         engines: ['twig', 'php']
+        hinclude_default_template: ":default:hinclude.html.twig"
```
Comment (phansys) [inline config/packages/framework.yaml:20]:
```suggestion
        hinclude_default_template: ':default:hinclude.html.twig'
```

File: config/packages/sonata_customer.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Enable Doctrine to map the provided entities
 sonata_customer:
     profile:
-        #template:     '@SonataCustomer/Profile/action.html.twig'
+        template:     '@SonataCustomer/Profile/action.html.twig'
```
Comment (phansys) [inline config/packages/sonata_customer.yml:4]:
```suggestion
        template: '@SonataCustomer/Profile/action.html.twig'
```

File: config/packages/sonata_twig.yaml [hunk 4]
```diff
@@ -2,6 +2,18 @@ sonata_twig:
...
+        - { type: 'sonata_product_success' }
+        - { type: 'sonata_user_success' }
+    error:
+      css_class: 'error'
+      types:
+        - { type: 'error' }
+        - { type: 'sonata_customer_error' }
+        - { type: 'sonata_flash_error' }
+        - { type: 'sonata_product_error' }
+        - { type: 'sonata_user_error' }
```
Comment (phansys) [inline config/packages/sonata_twig.yaml:19]: Missing LF at EOF.

**Review by phansys — CHANGES_REQUESTED** [review]

File: composer.json [hunk 5]
```diff
@@ -41,6 +41,7 @@
         "sonata-project/page-bundle": "^3.18",
         "sonata-project/seo-bundle": "^2.11",
         "sonata-project/timeline-bundle": "^3.6",
+        "sonata-project/twig-extensions": "1.4.1",
```
Comment (phansys) [inline composer.json:44]: Do we really need an exact version here? Can't we trust on semver?

Comment (wbloszyk) [reply composer.json]: This change is require. 1.4.0 will be not working.
https://github.com/sonata-project/twig-extensions/pull/103/files

Fixed to "^1.4.1"

**Review by phansys — APPROVED** [review]

**Pull request merged**